src/tests/tablet/schedule/AMeetingCanBeDeleted.java

```diff
@@ -0,0 +1,51 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import framework.common.MeetingMethods;
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC13: Verify a user can remove meetings
```

Comment:
be carefull with the titles of the TCs this is wrong ....Verify that a user can remove a meeting